it/v6/foreignPropertyBsas/retrieve/def1/Def1_RetrieveForeignPropertyBsasISpec.scala

```diff
@@ -187,18 +178,18 @@ class Def1_RetrieveForeignPropertyBsasISpec extends IntegrationBaseSpec {
   }
 
   private trait NonTysTest extends Test {
-    def taxYear: Option[String] = None
+    def taxYear: String = "2019-20"
 
-    def mtdUrl: String = s"/$nino/foreign-property/$calculationId"
+    def mtdUrl: String = s"/$nino/foreign-property/$calculationId/$taxYear"
 
     def downstreamUrl: String = s"/income-tax/adjustable-summary-calculation/$nino/$calculationId"
 
   }
 
   private trait TysTest extends Test {
-    def taxYear: Option[String] = Some("2023-24")
+    def taxYear: String = "2023-24"
 
-    def mtdUrl: String = s"/$nino/foreign-property/$calculationId"
+    def mtdUrl: String = s"/$nino/foreign-property/$calculationId/$taxYear"
```

Comment:
The buildRequest hard-codes the URL so this (and the non-TYS counterpart) are not required I think. Prob also in Def2 and maybe elsewhere.)